cluster/saltbase/salt/kubelet/default

```diff
@@ -41,6 +41,12 @@
 {% endif -%}
 
 {% set config = "--config=/etc/kubernetes/manifests" -%}
+
+{% set manifest_url = "" -%}
+{% if grains['roles'][0] == 'kubernetes-master' and grains.cloud in ['gce'] -%}
```

Comment:
Well, the daemon controller will handle the node case in the very near future, and adding it to the nodes adds another edge-feature that isn't worth properly supporting given that there's not much use for it once the daemon controller exists.

That said, reducing special cases is a reasonable goal, so I'm happy to go that way if you'd prefer.